plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryConfig.java

```diff
@@ -47,6 +48,7 @@
     private int maxReadRowsRetries = DEFAULT_MAX_READ_ROWS_RETRIES;
     private boolean caseInsensitiveNameMatching;
     private Duration caseInsensitiveNameMatchingCacheTtl = new Duration(1, MINUTES);
+    private Duration viewsCacheTtl = new Duration(15, TimeUnit.MINUTES);
```

Comment:
nit: `MINUTES` is already imported at L33. Please use it instead of `TimeUnit.MINUTES`.

Reply:
Ahh..my bad! I'll update.

Reply:
Done.